yt_dlp/extractor/porn91.py

```diff
@@ -32,29 +68,40 @@ def _real_extract(self, url):
         if '作为游客，你每天只可观看10个视频' in webpage:
             raise ExtractorError('91 Porn says: Daily limit 10 videos exceeded', expected=True)
 
-        title = self._search_regex(
-            r'<div id="viewvideo-title">([^<]+)</div>', webpage, 'title')
-        title = title.replace('\n', '')
+        title = self._html_extract_title(webpage)
+        title = remove_end(title.replace('\n', ''), 'Chinese homemade video').strip()
 
         video_link_url = self._search_regex(
-            r'<textarea[^>]+id=["\']fm-video_link[^>]+>([^<]+)</textarea>',
+            r'document\.write\(\s*strencode2\s*\(\s*((?:"[^"]+\")|(?:\'[^\']+\'))\s*\)\s*\)',
             webpage, 'video link')
-        videopage = self._download_webpage(video_link_url, video_id)
+        video_link_url = self._search_regex(
+            r"src=\'([^\']+)\'", urllib.parse.unquote(video_link_url), 'unquoted video link')
+
+        release_date = self._search_regex(
+            r'<span\s+class=["\']title-yakov["\']>(\d{4}-\d{2}-\d{2})</span>', webpage, 'timestamp', fatal=False)
+        release_date = release_date.replace('-', '') if release_date else None
 
-        info_dict = self._parse_html5_media_entries(url, videopage, video_id)[0]
+        description = self._search_regex(
+            r'<span\s+class=["\']more title["\']>([^<]*)<br', webpage, 'description', fatal=False).strip()
```

Comment:
```suggestion
        description = self._html_search_regex(
            r'<span\s+class=["\']more title["\']>([^<]+)<br', webpage, 'description', fatal=False)
```
You realize the .strip negates the whole point of fatal=False, right?

Reply:
Oops